login_KA.feature

```diff
@@ -0,0 +1,144 @@
+Feature: log in Scoring 
+Background: 
+    Given installed and launched VPN on local PC
+    And a Gmail account
+    And user open https://qa.ddso-spot.quantori.com page
+    And user clicks "<signed in with google>" button
+
+Scenario: 1_1.1 Log in with valid login and password which were saved
+   Given user has login and password for gmail account saved on the device
+   When user clicks on this gmail account
+   Then user is logged in
+   And user sees the main page
+
+   Examples:
+   | email                          |  | password       |
+   | anna.kalinina@quantoria.cademy |  | right_password |
+
+Scenario: 1_1.2 Log in with valid login and password
+   Given user has login and password for gmail account 
+   And user chooses "<change the account>" button
```

Comment:
'<>' should be deleted since it cannot be filled from 'Examples'. And the same for the next lines

Reply:
Corrected!